samples/tfm_integration/psa_level_1/README.rst

```diff
@@ -134,16 +134,25 @@ Build Zephyr with a non-secure configuration:
 
       $ west build -p -b lpcxpresso55s69_ns samples/tfm_integration/psa_level_1/ --
 
-Next we need to manually flash the secure (``tfm_s.hex``)
-and non-secure (``zephyr.hex``) images wth a J-Link as follows:
+Make sure your board is set up with :ref:`lpclink2-jlink-onboard-debug-probe`.
+Next we need to manually flash the resulting image (``tfm_merged.bin``) with a
+J-Link as follows:
```

Comment:
```suggestion
Make sure your board is set up with :ref:`lpclink2-jlink-onboard-debug-probe`,
since this isn't the debug interface boards ship with from the factory:

Next we either manually flash the resulting image (``tfm_merged.bin``) with a
J-Link as follows:
```